docs/src/pages/components/cdk-components/sidenav/Sidenav.jsx

```diff
@@ -21,7 +21,11 @@ function Sidenav() {
         <SidenavPropsTable />
       </Section>
       <Section>
-        <DxcHeading level={3} text="Children" margin={{ bottom: "small" }} />
+        <DxcHeading
+          level={3}
+          text="Compound components"
+          margin={{ bottom: "small" }}
+        />
         <p>
```

Comment:
I don't think this is clear in the documentation. I would keep children (because they are children and should be documented like this), but I would put `DxcSidenav.Title` above, outside of children.

Reply:
I thought about that and having DxcSidenav.Title above the children it would be in a different hierarchy unless we add another title at the same level as the children title.

Reply:
Yes, when I say that DxcSidenav.Title should not be inside Children, I refer that it is not a children. But we can put a title for it.

Reply:
Old docs not updated.